Apps/Examples/Examples/Controllers/ExampleTableViewController.swift

```diff
@@ -94,8 +94,11 @@ extension ExampleTableViewController {
           example = examples[indexPath.row]
         }
 
-        let cell = UITableViewCell(style: .default, reuseIdentifier: "reuseIdentifier")
+        let cell = UITableViewCell(style: .subtitle, reuseIdentifier: "reuseIdentifier")
         cell.textLabel?.text = example.title
+        cell.textLabel?.numberOfLines = 2
+        cell.detailTextLabel?.text = example.description.trimmingCharacters(in: .whitespacesAndNewlines)
+        cell.detailTextLabel?.numberOfLines = 2
```

Comment:
Initially this was an aid during development, that turned out to be quite handy. Android uses a similar style(with a subtitle) as well.

Reply:
Can we consider adding:

```
        if #available(iOS 13.0, *) {
            cell.detailTextLabel?.textColor = .secondaryLabel
        } else {
            cell.detailTextLabel?.textColor = .lightGray
        }
```

| Before | After |
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-03-29 at 14 35 06](https://user-images.githubusercontent.com/205658/160693181-a4f4db68-a2d8-4859-b7c8-19bab2034fe2.png) | ![Simulator Screen Shot - iPhone 13 - 2022-03-29 at 14 34 25](https://user-images.githubusercontent.com/205658/160693206-ada40dc8-7726-48f6-8cfc-9fb8b7681ea8.png) |
| ![Simulator Screen Shot - iPhone 13 - 2022-03-29 at 14 35 09](https://user-images.githubusercontent.com/205658/160693185-48834390-b291-479b-ad4d-b1db2e3279d0.png) | ![Simulator Screen Shot - iPhone 13 - 2022-03-29 at 14 34 16](https://user-images.githubusercontent.com/205658/160693201-fa26c19a-1a10-4570-b8f0-fd74a2d12a1a.png) |

Feels a little easier to scan to my eyes